main.go

```diff
@@ -93,9 +83,40 @@ func main() {
 	log.Infof("\nChangelog:")
 	log.Printf(changelog)
 
+	envmanConfigs, err := envman.GetConfigs()
+	if err != nil {
+		failf("Failed to load envman configs, error: %s", err)
+	}
+
+	if envmanConfigs.EnvBytesLimitInKB > 0 {
+		envBytesLimit := envmanConfigs.EnvBytesLimitInKB * 1024
+
+		if len(changelog) > envBytesLimit {
+			log.Warnf("Changelog content exceeds the maximum allowed size to set in an environment variable. (%dKB)", envmanConfigs.EnvBytesLimitInKB)
+			log.Warnf("The changelog's content will be trimmed to fit the maximum allowed size.")
+			log.Warnf("It is possible to modify the limit by following this guide: https://devcenter.bitrise.io/tips-and-tricks/increasing-the-size-limit-of-env-vars")
+			changelog = changelog[:envBytesLimit-4] + "\n..."
+		}
+	}
+
 	if err := tools.ExportEnvironmentWithEnvman("BITRISE_CHANGELOG", changelog); err != nil {
```

Comment:
Would reccommend adding changelog_pth as an output, so users can see that there is an alternative.

Reply:
Changelog path is an input 🤔

Reply:
Adding it as an output makes it possible to be used in subsequent steps. The contents is an output of this step after all, so it makes sense I think as an output.

Reply:
Discussed with the PM and we will not export the file path in the step, as the path is available in other steps and reusable.